docker/beta/spark_reader/consul-template-spark-reader.conf

```diff
@@ -0,0 +1,17 @@
+template {
+    source = "/code/listenbrainz/consul_config.py.ctmpl"
+    destination = "/code/listenbrainz/listenbrainz/config.py"
+}
+
+template {
+    source = "/code/listenbrainz/admin/config.sh.ctmpl"
+    destination = "/code/listenbrainz/admin/config.sh"
```

Comment:
I'm not sure which services require config.sh. It seems like at least some of the spark code requires it, as well as the cron jobs. See other note about rendering templates for cron.

Reply:
I'm pretty sure config.sh is only required by the cron jobs, no spark code runs with consul, so it should be okay.

Reply:
I found a reference to it in https://github.com/metabrainz/listenbrainz-server/blob/a464312a452375a79a3693a49011d7b13e71a4a9/docker/start-request-consumer-container.sh#L5, but I don't know what this file is used for

Reply:
that is a different config.sh, sadly 🙈